src/documents/components/ReviewFiles/ReviewFileElement.js

```diff
@@ -0,0 +1,139 @@
+import {Checkbox, Icon, Popup, Table} from 'semantic-ui-react';
+import React, {useEffect, useState} from 'react';
+import {fileLatestDate, fileSortedVersions} from '../../utilities';
+
+import FileTags from '../FileDetail/FileTags';
+import KfId from '../../../components/StudyList/KfId';
+import Markdown from 'react-markdown';
+import TimeAgo from 'react-timeago';
+import {fileTypeDetail} from '../../../common/enums';
+import {longDate} from '../../../common/dateUtils';
+import {withRouter} from 'react-router-dom';
+
+const useRecentlyUpdated = (latestDate, fileId) => {
+  const [justUpdated, setJustUpdated] = useState(false);
+
+  useEffect(() => {
+    let FIVE_MIN = 300000;
+    let docs = sessionStorage.getItem('kf_updated_docs');
```

Comment:
Is `kf_updated_docs` ever set anywhere?

Reply:
It is set in src/documents/modals/UploadWizard/UploadWizard.js 
https://github.com/kids-first/kf-ui-data-tracker/blob/630807391403fa11ae93fcd00716a332fd2162c6/src/documents/modals/UploadWizard/UploadWizard.js#L108